src/lib/cache/cacheManager.ts

```diff
@@ -0,0 +1,356 @@
+import { Cache, caching } from 'cache-manager';
+import { getConn, IPrefixCommand, IPrefixCommandCategory, IPrefixCommandChannelDefaultVersion, IPrefixCommandVersion, Logger, PrefixCommand, PrefixCommandCategory, PrefixCommandChannelDefaultVersion, PrefixCommandVersion } from '../index';
+
+let inMemoryCache: Cache;
+const cacheSize = 10000;
+const cacheRefreshInterval = process.env.CACHE_REFRESH_INTERVAL ? Number(process.env.CACHE_REFRESH_INTERVAL) : 1800;
+const cacheTTL = cacheRefreshInterval * 2 * 1000;
+
+/**
+ * Cache Prefixes
+ */
+
+export const memoryCachePrefixCommand = 'PF_COMMAND';
+export const memoryCachePrefixVersion = 'PF_VERSION';
+export const memoryCachePrefixCategory = 'PF_CATEGORY';
+export const memoryCachePrefixChannelDefaultVersion = 'PF_CHANNEL_VERSION';
```

Comment:
```suggestion
export enum MemoryCachePrefix {
    COMMAND = 'PF_COMMAND',
    VERSION = 'PR_VERSION',
    CATEGORY = 'PR_CATEGORY',
    CHANNEL_DEFAULT_VERSION = 'PF_CHANNEL_VERSION',
}
```
I would use an enum here to better group the static exports together. Imo it also helps with readability. 

Apart from that, global static constants are usually written in screaming snake case. I only noticed this when I wondering where those variables were defined while reviewing another file. I'd change that for readability.

Reply:
Awesome suggestion, thanks!